src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/config/ManageRecallsApiExceptionHandler.kt

```diff
@@ -101,6 +111,10 @@ class ManageRecallsApiExceptionHandler {
 
 data class ErrorResponse(val status: HttpStatus, val message: String?)
 
+data class FileError(val category: DocumentCategory, val fileName: FileName, val error: String?)
```

Comment:
Should this be `FileException`?

Reply:
I don't think so; it is a list entry in a MultiFileError.